packages/block-library/src/cover/block.json

```diff
@@ -39,8 +38,7 @@
 			"type": "string"
 		},
 		"backgroundType": {
-			"type": "string",
-			"default": "image"
+			"type": "string"
```

Comment:
Thanks for looking at this @WunderBart 🎉

For me turning off the default here causes any blocks created prior to this patch to break with validation errors - which is why "backgroundType":"image" needed to be added to all the fixtures I think. With a default set the attribute is never serialized into the block content.

I think if we went with this approach we would need to add a new deprecation that serialized the backgroundType into the content for all existing cover blocks.